math/examples/riscv_tests/beaglev_fire/sin_performance/readme.md

```diff
@@ -0,0 +1,16 @@
+## sin_performance
```

Comment:
Should we move this reame.md one level up in the directory hierarchy to cover both boards: icicleKit and BFV ?
Maybe refactor the hierarchy to be:
math/examples/riscv_tests/sin_performance/beaglev_fire
math/examples/riscv_tests/sin_performance/icicle_kit
math/examples/riscv_tests/sin_performance/sources
math/examples/riscv_tests/sin_performance/readme.md